libs/stac/docs/conf.py

```diff
@@ -0,0 +1,101 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+from odc.stac._version import __version__ as _odc_stac_version
+#print(sys.path)
+
+
+# -- Project information -----------------------------------------------------
+
+project = "STAC.load"
```

Comment:
What's the rationale behind this name, vs the PyPI name of `odc-stac`?

Reply:
no good reason, it also lives on rtd under that name, but we should probably move it to `odc.stac` instead.

Reply:
:+1: for trying to keep the same name as much as possible (`odc-stac` for urls and such, `odc.stac` for python imports).

Reply:
```suggestion
project = "odc-stac"
```